src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -382,5 +397,12 @@ private static string GetSize(long length)
             {
                 Argument = new Argument<string>(name: "clreventlevel", getDefaultValue: () => string.Empty)
             };
+        private static Option DiagnosticPortOption() =>
+            new Option(
+                alias: "--port",
```

Comment:
I would suggest "--diagnostic-port" instead of "--port" since that can be ambiguous.

Reply:
dotnet-monitor is already using "--diagnostic-port" so using the same name would be consistent across tools.